code/game/objects/items/grenades/plastic.dm

```diff
@@ -105,6 +105,11 @@
 		return
 	if(ismob(AM) && !can_attach_mob)
 		return
+	if(istype(AM, /obj/item/valentine))
```

Comment:
There's seriously no better way to do this? Preferably on the valentine item?

Reply:
No, I thought there would be some c4act proc, but on planting the c4 just moves to null space and then blows the item up after x seconds

Reply:
no